src/main/java/binnie/extrabees/gui/database/ControlProductsItem.java

```diff
@@ -24,7 +24,7 @@ public ControlProductsItem(ControlList<ControlProductsBox.Product> controlList,
 		}
 
 		item.setItemStack(value.item);
-		float time = (int) (55000.0 / value.chance);
+		float time = (int) (550.0 / value.chance);
```

Comment:
Because production time was displayed 100 times more. 15.2 hours instead 4.2 minutes for example.
I didn't begin to see, why this happened :)